client/src/Movies/Imdb.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+//imports a random movie or TV show from IMDB api...
+class Imdb extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      movieList: []
+    };
+  }
+  componentDidMount = () => {
+    fetch(
+      "https://www.omdbapi.com/?apikey=8894cd6c&plot=full&i=tt0" +
+        `${Math.floor(Math.random() * (999999 - 100000) + 100000)}`
+    )
+      .then(res => res.json())
+      .then(result => {
+        this.setState({
+          movieList: new Array(result)
+        });
+      });
```

Comment:
```suggestion
      }).catch(err => console.error(err));
```
If there is a network error or something, this catch should fire rather than any of the then blocks, so we still need a .catch in case of network errors.